src/wix/WixToolset.Converters/ConversionLab.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+
+namespace WixToolset.Converters
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Xml.Linq;
+
+    internal class ConversionLab : IDisposable
```

Comment:
What does `ConversionLab` mean? I think it needs a clearer name.

Reply:
It's short for Conversion Laboratory. I'm open to suggestions for a better paradigm.